app/mailers/sac_cas/event/participation_mailer.rb

```diff
@@ -7,23 +7,57 @@
 
 module SacCas::Event::ParticipationMailer
   extend ActiveSupport::Concern
+  include EventMailer
 
   CONTENT_REJECTED_PARTICIPATION = "event_participation_rejected"
+  CONTENT_SUMMON = "event_participation_summon"
 
   def reject(participation)
+    compose_email(participation, CONTENT_REJECTED_PARTICIPATION)
+  end
+
+  def summon(participation)
+    compose_email(participation, CONTENT_SUMMON)
+  end
+
+  private
+
+  def compose_email(participation, content_key)
     @participation = participation
+    @course = @participation.event
     person = @participation.person
+    headers = {bcc: @course.groups.first.course_admin_email}
+    locales = @course.language.split("_")
 
-    compose(person, CONTENT_REJECTED_PARTICIPATION)
+    compose(person, content_key, nil, headers, locales)
+  end
+
+  def compose(recipients, content_key, values = nil, headers = {}, locales = [])
+    return if recipients.blank?
+
+    values = if values
+      values.merge(
+        "event-details" => event_details,
+        "application-url" => link_to(participation_url)
+      )
+    else
+      values_for_placeholders(content_key)
+    end
+
+    custom_content_mail(recipients, content_key, values, headers, locales)
   end
```

Comment:
```suggestion
    compose(person, content_key, headers, locales)
  end
```

https://github.com/hitobito/hitobito_sac_cas/blob/c12900312dfff8c73e824a604ddab028e1a5020d/app/mailers/sac_cas/application_mailer.rb#L11

Reply:
und statt `"application-url" => link_to(participation_url)` würde ich das in ein `def placeholder_` tun und den `compose` vom sac_cas/application_mailer verwenden